certificatetprtest/service.go

```diff
@@ -0,0 +1,18 @@
+package certificatetprtest
```

Comment:
Might make sense to make this a recorder, depending on your further needs. Like in `httptest.ResponseRecorder` https://golang.org/pkg/net/http/httptest/#ResponseRecorder